src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -140,21 +139,17 @@ def post(self, request, project):
         report['event_id'] = report['event_id'].lower()
         report['project'] = project
 
+        event = SnubaEvent.objects.from_event_id(report['event_id'], project.id)
+
         # TODO(dcramer): we should probably create the user if they dont
         # exist, and ideally we'd also associate that with the event
-        euser = self.find_event_user(report)
+        euser = self.find_event_user(report, event)
         if euser and not euser.name and report['name']:
             euser.update(name=report['name'])
         if euser:
             report['event_user_id'] = euser.id
 
-        event = Event.objects.from_event_id(report['event_id'], project.id)
-        if not event:
-            try:
-                report['group'] = Group.objects.from_event_id(project, report['event_id'])
```

Comment:
`Group.objects.from_event_id` is now basically the same query in Snuba as `SnubaEvent.objects.from_event_id` so we don't need to try both